### PR TITLE
Move useHelm3 switch to end until we want it so visible.

### PR DESCRIPTION
**Pull request by absoludity**
### Description of the change
Just noticed this when upgrading Kubeapps itself:

useHelm3-first-option-in-values

Not a big deal, but we probably don't want it to be the first thing a user sees until it actually works.

### CODE REVIEW

File: chart/kubeapps/values.yaml [hunk 1]
```diff
@@ -583,3 +580,7 @@ authProxy:
     requests:
       cpu: 25m
       memory: 32Mi
+
+# true for Helm 3; false for Helm 2:
+useHelm3: false # DOES NOT WORK YET; will act as a feature flag later on when we have Helm 3 support.
```
Comment (andresmgot) [inline chart/kubeapps/values.yaml]: I would move the `DOES NOT WORK YET` ( or something similar to the title instead of being an inline comment)

Comment (SimonAlling) [reply chart/kubeapps/values.yaml]: Do you mean like this?

```diff
# true for Helm 3; false for Helm 2:
-useHelm3: false # DOES NOT WORK YET; will act as a feature flag later on when we have Helm 3 support.
+# DOES NOT WORK YET; will act as a feature flag later on when we have Helm 3 support.
+useHelm3: false
```

Comment (SimonAlling) [reply chart/kubeapps/values.yaml]: Hm ... that feels pretty inline as well when I think about it.

Comment (absoludity) [reply chart/kubeapps/values.yaml]: > Hm ... that feels pretty inline as well when I think about it.

I think he meant simply not having the comment on the same line after the text.

Anyway, updated and expanded the comment a bit).

**Comment by absoludity** [comment]
Side point: the timing of this was interesting: right after I noticed this while upgrading Kubeapps and created this issue, I saw that a user had created #1375 after they tried the flag too.

**Pull request merged**